pkg/scheduler/algorithm/priorities/util/non_zero.go

```diff
@@ -16,7 +16,7 @@ limitations under the License.
 
 package util
 
-import "k8s.io/api/core/v1"
+import v1 "k8s.io/api/core/v1"
```

Comment:
Oh no!!! I made the classic VSCode mistake.

Reply:
what's the harm? if many editors add it by default, why not just make it part of the format :)